storage_controller/src/http.rs

```diff
@@ -1305,6 +1305,35 @@ async fn handle_upsert_safekeeper(mut req: Request<Body>) -> Result<Response<Bod
         .unwrap())
 }
 
+/// Sets the scheduling policy of the specified safekeeper
+async fn handle_safekeeper_scheduling_policy(
+    mut req: Request<Body>,
+) -> Result<Response<Body>, ApiError> {
+    check_permissions(&req, Scope::Infra)?;
```

Comment:
Will Infra scripts set the scheduling policy?  For the similar `handle_node_configure` for pageserver we require admin scope.

The way I think of it is:
- Infra == scripts, things that happen regularly in deploys
- Admin == humans, for on-call

Reply:
Done